src/main/java/com/celements/web/plugin/api/CelementsWebPluginApi.java

```diff
@@ -685,24 +622,6 @@ public String navReorderSave(String fullName, String structureJSON) {
         fullName), structureJSON);
   }
 
-  /**
-   * @deprecated since 2.34.0 instead use
-   *             {@link LayoutScriptService #layoutExists(SpaceReference)}
-   */
-  @Deprecated
-  public boolean layoutExists(String layoutSpaceName) {
-    return this.layoutExists(getWebUtilsService().resolveSpaceReference(layoutSpaceName));
-  }
-
-  /**
-   * @deprecated since 2.34.0 instead use
-   *             {@link LayoutScriptService #layoutExists(SpaceReference)}
-   */
-  @Deprecated
-  public boolean layoutExists(SpaceReference layoutSpaceRef) {
-    return getLayoutScriptService().layoutExists(layoutSpaceRef);
-  }
-
```

Comment:
still used in at least celements-photo/web

Reply:
- https://github.com/celements/celements-image/pull/82
- https://github.com/celements/celements-web/pull/458
- https://github.com/progonline/progon-web/pull/686